homeassistant/components/zabbix/manifest.json

```diff
@@ -4,7 +4,7 @@
   "codeowners": [],
```

Comment:
Are you willing to maintain this integration?
If yes please add yourself as code owner

Reply:
Sure. There is a lot of work that remains here that I can finish and maintain.